### DISCUSSION TITLE
Apache nifi processor

### DISCUSSION THREAD

**Discussion by Praveenstein** [apache/iotdb#8782] [Q&A] [answered]
Do we have any official nifi processor coming to support iotdb??

**Comment by xuanronaldo** [comment, marked as answer]
We have a `PutIoTDB` processor to insert data from NiFi, and the PR of the code has been commited. But it has not been merged yet. Here is the link: https://github.com/apache/nifi/pull/6416

**Reply by Praveenstein** [reply]
thanks for the reply
